docs/GPSACP.md

```diff
@@ -0,0 +1,66 @@
+# GPSACP message
+
+GPSACP messages are defined in [WTP - SCG Reference Points](https://www.hochbahn.de/hochbahn/wcm/connect/de/1d8945fd-174c-447a-85a5-4a8424406b5a/Lieferung%2Bvon%2BWLAN%2BRoutern%2BAB%2B17.12.2016.pdf?MOD=AJPERES&CACHEID=ROOTWORKSPACE.Z18_JH8I1JC0L05M10AEB6TSP430A1-1d8945fd-174c-447a-85a5-4a8424406b5a-lLovXv0) can be sent within WTP Event Requests.
+
+## Definition GPS Last Acquired Position
+The GPS Last Acquired Position contains the output of the AT$GPSACP command from an WWAN Modem.
+
+### Format
+
+```
+0                   1                   2                   3
+0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                         Timestamp                             |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|    WWAN Id    |       GPSATC ....
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+
+**Vendor Id** 18681 (Travelping GmbH)
+
+**Type** 15
+
+**Length** >= 5
+
+**Timestamp** The WTP’s time when the meassurment was recorded
+
+**WWAN Id** WWAN Interface Id, used to differentiate between multiple WWAN interfaces, MUST be between
+one (1) and 16.
+
+**GPSATC** The full output (including the starting $GPSATC) of the AT$GPSATC command the WWAN Interface:
+
+```
+$GPSACP:<UTC>,<latitude>,<longitude>,<hdop>,<altitude>,<fix>,<cog>,<spkm>,<spkn>,<date>,<nsat> 
+
```

Comment:
The format is support by several m2m LTE moduls. It my not be the original, but this format was taken from Telit LTE modules with GPS support (e.g.: http://www.telit.com/fileadmin/user_upload/products/Downloads/3G/xe910/Telit_HE910_A-GPS_Application_Note_r4.pdf)